pibooth/printer.py

```diff
@@ -101,9 +108,9 @@ def print_file(self, filename, copies=1):
                 # are the one necessary to render several pictures on same page.
                 factory.set_margin(2)
                 factory.save(fp.name)
-                self._conn.printFile(self.name, fp.name, osp.basename(filename), {})
+                self._conn.printFile(self.name, fp.name, osp.basename(filename), self.options)
         else:
-            self._conn.printFile(self.name, filename, osp.basename(filename), {})
+            self._conn.printFile(self.name, filename, osp.basename(filename), self.options)
         LOGGER.debug("File '%s' sent to the printer", filename)
```

Comment:
@anxuae Can we add the print options to the log output too e.g. `LOGGER.debug("File '%s' sent to the printer with options %s", filename, str(self.options))`

Reply:
added